### DISCUSSION TITLE
Is the new `Searching for members 🔍` feature replacing how `Go to File...` used to work?

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#1730] [IBM i]
Since the feature update for `Go to File...` was implemented I have been unable to open a specific member reference.  If I type string like `lib/file/member.ext` nothing happens.  Pressing Enter will not cause the tool to open the member.  I have to remove the extension, type an asterisk (*) and wait for the query to finish (which on our 7.5 sometimes takes a long time).  Once the action of the search has completed for a particular source file the search is usually quicker to return results. 

So am I not able to specify an specific member path anymore?

Thanks, Matt

**Comment by worksofliam** [comment]
Should be able to do both. I can take a look.

**Comment by chrjorgensen** [comment]
@m-tyler There were indeed some errors in the code after implementing searching for members in Go To file.
I have created an issue #1732 from this discussion and a fix PR #1733 for this.

I will close this discussion - further communication should be made in #1732.
